presets.c

```diff
@@ -26,6 +26,10 @@
 
 #include "v4l2-request-test.h"
 
+/*
```

Comment:
Well, we already have the description / attribution fields for this info, so I don't really see the point of duplicating it as a comment.

Reply:
Probably just a helper comment (for me) to get into the code. Sure, it is declared in the stucture.